src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,385 @@
+| ------------------- | --------------------- | --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| SourceType          | RECOMMENDED           | [number][] or `mixed` | Type of source. Preferably a specific model/part number is supplied. This is a freeform description, but the following keywords are suggested: LED, LASER, VCSEL. If individual channels have different SourceTypes, then the field here should be specified as `mixed`  and this column should be included in `optodes.tsv`.                                                          |
+| DetectorType        | RECOMMENDED           | [number][] or `mixed` | Type of detector. This is a free form description with the following suggested terms: SiPD, APD. Preferably a specific model/part number is supplied. If individual channels have different DetectorTypes, then the field here should be specified as `mixed`  and this column should be included in `optodes.tsv`.                                                                    |
+| ShortChannelCount   | RECOMMENDED           | [number][]            | The number of short channels. 0 indicates no short channels.                                                                                                                                                                                                                                                                                                                           |
+| NIRSPlacementScheme | RECOMMENDED           | [string][]            | Placement scheme of NIRS optodes. Either the name of a standardized placement system (for example, "10-20") or a list of standardized position names (for example, ["Cz", "Pz"]). This field should only be used if a cap was not used. If a standard cap was used, then it should be specified in CapManufacturer and CapManufacturersModelName and this field should be set to “n/a” |
+
+Example:
+
+```JSON
+{
+  "TaskName":"visual",
+  "InstitutionName":"Macquarie University. Australian Hearing Hub",
+  "InstitutionAddress":"6 University Ave, Macquarie University NSW 2109 Australia",
+  "Manufacturer":"NIRx",
+  "ManufacturersModelName":"NIRScout",
+  "TaskDescription":"visual gratings and noise patterns",
+  "Instructions":"look at the dot in the center of the screen and press the button when it changes color",
+  "SamplingFrequency":3.7,
+  "NIRSChannelCount":56,
+  "NIRSSourceOptodeCount":16,
+  "NIRSDetectorOptodeCount":16,
+  "ACCELChannelCount":0,
+  "SoftwareFilters":"n/a",
+  "RecordingDuration":233.639,
+  "DCOffsetCorrection":0,
```

Comment:
the field DCOffsetCorrection is used in the example, but not defined in any table

Reply:
the specified value is `0`, but I wonder whether it should be a JSON Boolean, i.e. `true` or `false`